### DISCUSSION TITLE
Changing Port on Peer.js Server for a Newbie to Node.js

### DISCUSSION THREAD

**Discussion by JAA17** [peers/peerjs#1208] [Q&A] [answered]
My webserver people at Winhost say I cannot use port 9000. I have asked if I can use any port with them:

https://forum.winhost.com/threads/peer-js-peer-server.30902/

If I can access ports at Winhost, what do I need to do with the Peer.js Server to change all the ports in the code to the new port? Can anyone give a newbie a step by step guide of what I need to do?

**EDIT:** I have found I need to do it on port 80. Can anyone help me to understand how this can be done?

**Comment by anpolimus** [comment, marked as answer]
Hi, @JAA17 .
Port id could be changed during initial run: 
`peerjs --port 30902 --key peerjs --path /myapp`
